pkg/cmd/operator/webhooks.go

```diff
@@ -230,10 +230,10 @@ func validate(ar *admissionv1.AdmissionReview) error {
 	case scyllav1.GroupVersion.WithResource("scyllaclusters"):
 		switch ar.Request.Operation {
 		case admissionv1.Create:
-			return validation.ValidateScyllaCluster(obj.(*scyllav1.ScyllaCluster))
+			return validation.ValidateScyllaCluster(obj.(*scyllav1.ScyllaCluster)).ToAggregate()
```

Comment:
maybe we should plumb it into the webhook and set `StatusDetails` with `apierrors.NewInvalid`